lib/issue_utils.dart

```diff
@@ -29,4 +29,43 @@ extension IssueUtils on Issue {
   }
 
   bool authorIsGoogler(Set<String> googlers) => googlers.contains(user?.login);
+
+  Issue close() {
```

Comment:
Does this method operate on the issue? Did you mean `clone` here?

Reply:
It's actually about getting a closed version of the issue - This should be more clear now.